data/conf/infrasim.yml

```diff
@@ -0,0 +1,60 @@
+---
+#This file is used as InfraSIM configuration file
+
+main:
+    #Supported vnode names:
+    #	quanta_d51
+    #	quanta_t41
+    #	dell_c6320
+    #	dell_r630
+    #	s2600kp
+    #	s2600tp
+    #	s2600wtt
+    node: quanta_d51

```

Comment:
I'll vote for what was implemented in infrasim-vxrail, those fields like:
emu_file: chassis/node3/node3.emu
smbios: chassis/node4/node4.bin

feel well if everything being under control. :-)